ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -364,7 +363,7 @@ - (NSString *) getBuildId {
     return self.getBuildId;
 }
 
-- (NSString *) getUniqueId {
```

Comment:
I would expect both flavors here and an internal method ? An internal method that does the work, One that is the getUniqueIdSync and is RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD calling the internal method directly, and one that is a regular RCT_EXPORT_METHOD that takes theh RCTPromiseResolveBlock and resolves by calling the internal method. isTablet or isEmulator could serve as a good reference

Reply:
See new methods. Internal method getUniqueId renamed to uniqueId. React methods getUniqueIdSync/getUniqueId added.